Dockerfile.ubi8

```diff
@@ -0,0 +1,21 @@
+# Build the manager binary
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
+
+# Copy in the go src
+WORKDIR /go/src/github.com/kserve/kserve
+COPY go.mod  go.mod
+COPY go.sum  go.sum
+
+RUN go mod download
+
+COPY cmd/    cmd/
+COPY pkg/    pkg/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOFLAGS=-mod=mod go build -a -o manager ./cmd/manager
+
+# Use distroless as minimal base image to package the manager binary
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6 
```

Comment:
use ubi8 micro.
Smaller, better for runtime.

same for the others.

Reply:
I agreed. It there is no specific reasons to use uni-minial, micro would be a good option :(https://www.redhat.com/en/blog/introduction-ubi-micro)